tests/robot-tests/tests/admin_and_public/bau/publish_data.robot

```diff
@@ -513,6 +513,7 @@ Add text block with link to a featured table to accordion section
     user clicks button    Insert    ${modal}
     user waits until modal is not visible    Insert featured table link
     user clicks button    Save & close    ${block}
+    sleep    0.5
```

Comment:
For some reason, a failure happens consistently here on your branch. Not entirely sure why this is, because you've not changed any of the admin code really :thinking:

I checked some of your dependency upgrades and couldn't see anything off really.

Regardless, adding this sleep seems to fix it even though it's a bit naff.